wbia/constants.py

```diff
@@ -132,12 +132,12 @@
 class PATH_NAMES(object):
     """ Path names for internal IBEIS database """
 
-    sqldb = '_wbia_database.sqlite3'
```

Comment:
Unfortunately this change is not without consequences since it'll cause all of the existing databases to fail to be found.  We will need to carefully build in a database update and support aliasing for database files/folders on disk with a fallback to the `_ibeis` prefix if `_wbia` has not been found

Reply:
This is a change from `_wbia_database.sqlite3` to `_ibeis_database.sqlite3` (not ibeis to wbia) because I'm trying to avoid dealing with the filename changes.  I'm doing this change because doctests were failing.

Reply:
Ahh, gotcha.  Perfect, I'll approve if it's ready